packages/bull-vault/test/e2e/FlashBullTestFork.t.sol

```diff
@@ -148,7 +148,7 @@ contract FlashBullTestFork is Test {
             wethToLend
         );
         assertEq(
-            IERC20(crabV2).balanceOf(address(bullStrategy)).sub(crabToBeMinted),
+            bullStrategy.getCrabBalance().sub(crabToBeMinted),
```

Comment:
we are comparing the balanceOf to the strategy storage variable compared to the expected crab amt

we should probably compare the balanceOfs and the local variables and ensure they are consistent with eachother, so more assertions to add here